app/routines/newflow/create_or_update_salesforce_lead.rb

```diff
@@ -45,8 +45,10 @@ def exec(user:)
         adoption_json = build_book_adoption_json_for_salesforce(user)
       end
 
-      # The user has not finished signing up
-      user.faculty_status = User::INCOMPLETE_SIGNUP unless user.is_profile_complete?
+      # The user has not finished signing up, check for confirmed faculty if they were verified by sheerid
+      if !user.is_profile_complete? && !user.faculty_status == :confirmed_faculty
```

Comment:
I was trying to do something like 
```
user.faculty_status.include?(:confirmed_faculty, :pending_sheerid, :rejected_by_sheerid)
```
but clearly that's not the correct ruby syntax, any advise on checking the state within an array? I'm sure I'm just doing something backwards

Reply:
`faculty_status` is a Rails enum which is slightly weird. You can assign symbols or strings but the method returns a string if I remember right. I think either of these should work:

`user.confirmed_faculty? || user.pending_sheerid? || user.rejected_by_sheerid?`

`['confirmed_faculty', 'pending_sheerid', 'rejected_by_sheerid'].include?(user.faculty_status)`